File: 1주차/NH/twoSum.js
function twoSum(nums, target) {
    for(let i=0; i<nums.length; i++){
        if(nums.includes(target-nums[i])){
            if(nums.indexOf(target-nums[i])==i)continue;
            return[i,nums.indexOf(target-nums[i])]
        }
    }
  }

File: 1주차/YH/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)) : 
            j = i + 1
            for j in range(i) : 
                if (nums[i] + nums[j] == target) : 
                    return [i, j]